js/layout-javascript/news-feed-code.js

```diff
@@ -174,8 +174,10 @@ DynamicList.prototype.attachObservers = function() {
       _this.hideFilterOverlay();
       _this.clearFilters();
     })
-    .on('click', '.hidden-filter-controls-filter', function() {
-
+    .on('click keydown', '.hidden-filter-controls-filter', function(event) {
+      if (!_this.Utils.accessibilityHelpers.isExecute(event)) {
+        return;
+      }
```

Comment:
An empty line of spacing before and after `{ }` unless its in the beginning of a `{ }` block.

@upplabs-alex-levchenko @sofiiakvasnevska Please make sure Upplabs reviews all the PRs and review notes correctly. This PR has taken too long to get right just because the usual coding standards are not being followed. We should not be picking it out another error like this in the next review.

```suggestion
      }

```